### DISCUSSION TITLE
Support more versions

### DISCUSSION THREAD

**Discussion by DotDebian** [stefvanschie/IF#75] [Proposed changes]
Might be cool to support 1.8 as a lot of servers are based upon this version to support higher ones. Also please write the supported versions on your repo as I just lost some time using it with a noclassdeffound at the end.

**Comment by stefvanschie** [comment]
We've discussed the supported versions before in #48 and the result of that was to drop the earlier supported 1.12 and 1.13. However, we can discuss this again and see if more people want to support those versions again as well as 1.8.

I have added the current supported versions to the readme like you mentioned, hopefully this saves others some time in the future.
